lib/smart_answer_flows/marriage-abroad/outcomes/outcome_ceremonies_in_denmark_when_residing_in_uk_or_denmark.govspeak.erb

```diff
@@ -62,6 +70,13 @@
 
   <%= render partial: 'names_on_documents_must_match.govspeak.erb' %>
 
+  <% if calculator.resident_of_ceremony_country? %>
```

Comment:
This change seems unrelated to this commit. Moving this code only affects outcomes where the user is a resident of Denmark and this commit is specifically about users that are resident in the UK.

Should it be part of efd5abfb7846589e687054b3a93c6a35d3c58478 or 52297aea298457e754b2768847c1809e345f1ccc instead?